remote/remote_state_gcs.go

```diff
@@ -426,6 +427,11 @@ func CreateGCSClient(gcsConfigRemote RemoteStateConfigGCS) (*storage.Client, err
 
 	if gcsConfigRemote.Credentials != "" {
 		opts = append(opts, option.WithCredentialsFile(gcsConfigRemote.Credentials))
+	} else if gcsConfigRemote.AccessToken != "" {
```

Comment:
I think will be helpful to have a test that will track if handling of `AccessToken` will continue to work in future

Reply:
done